mimic/core.py

```diff
@@ -161,9 +189,14 @@ def entries_for_tenant(self, tenant_id, prefix_map, base_uri):
         for service_id, api in self._uuid_to_api_external.items():
             for entry in api.catalog_entries(tenant_id):
                 for endpoint in entry.endpoints:
-                    prefix_map[endpoint] = api.uri_for_service(
-                        endpoint.region, service_id
-                    )
+                    try:
+                        prefix_map[endpoint] = api.uri_for_service(
+                            endpoint.region, service_id
+                        )
+                    except IndexError:  # pragma:nocover
+                        # ignore APIs that don't support the given region
+                        pass
```

Comment:
@glyph I tried to create a test for the above try/except and then remove the pragma:nocover, but I'm not sure exactly how since it requires calling api.uri_for_service() with a different region than expected. I believe the try/except is necessary; but could be wrong. I'd love to write a test if possible. Advice/tips/etc appreciated.